CONTRIBUTING.md

```diff
@@ -27,7 +27,7 @@ In this guide we're going to go through the steps for each kind of contribution,
 <a name="-before-you-start"></a>
 ## 🏁 Before you Start
 
-Make sure you've read through our [README](https://github.com/jina-ai/jina), [Jina 101](./101/README.html), and [example tutorials](https://github.com/jina-ai/examples) so you have a good understanding of what Jina is and how it works.
+Make sure you've read through our [README](https://github.com/jina-ai/jina), [Jina 101](https://github.com/jina-ai/jina/tree/master/docs/chapters/101), and [example tutorials](https://github.com/jina-ai/examples) so you have a good understanding of what Jina is and how it works.
```

Comment:
```suggestion
Make sure you've read through our [README](https://github.com/jina-ai/jina), [Jina 101](https://docs.jina.ai/chapters/101/index.html), and [example tutorials](https://github.com/jina-ai/examples) so you have a good understanding of what Jina is and how it works.
```

Reply:
@nan-wang It makes sense. 
Taking this into consideration, I think we should change: https://github.com/jina-ai/examples/blob/master/README.md as well.
![image](https://user-images.githubusercontent.com/22430520/103752154-9df5ee00-5009-11eb-9bb9-4e125761fb49.png)